src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react';
+import { People } from '../components/People';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+
+export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+  const [people, setPeople] = useState<Person[] | null>(null);
+
+  useEffect(() => {
+    const fetchPeople = async () => {
+      try {
+        setIsLoading(true);
+        setError(null);
+
+        const response = await fetch(
+          'https://mate-academy.github.io/react_people-table/api/people.json',
+        );
+
+        if (!response.ok) {
+          throw new Error('Failed to fetch people');
+        }
+
+        const data = await response.json();
+
+        setTimeout(() => {
+          setPeople(data);
+        }, 500);
+      } catch (err) {
+        setError('Something went wrong');
```

Comment:
Consider logging the actual error message `err` to the console for debugging purposes. This can help identify specific issues during the fetch operation.